modules/test/playwright/pages/document-library-web/DocumentLibraryPage.ts

```diff
@@ -349,6 +323,20 @@ export class DocumentLibraryPage {
 		await fileEntryCheckbox.check();
 	}
 
+	async selectFolder(folderName: string) {
+		const folderCheckbox = this.page
+			.locator(`label:has-text('${folderName}')`)
+			.getByRole('checkbox');
+
+		if (await folderCheckbox.isHidden()) {
```

Comment:
Just wondering, why do you need to search for a folder? Maybe because it could be in a different page (pagination)?

Reply:
hey, its basically a carbon copy of selectFileEntry but with the css adjusted. I haven't given it a thought about the search part but during tests runs I could see that sometimes playwright falls back to a search. Maybe because of some loading condition.